src/Apps/Artwork/Components/ImageCarousel.tsx

```diff
@@ -177,74 +177,116 @@ export class ImageCarousel extends React.Component<
-                  />
-                )}
-                {xs && <Spacer />}
-                <ActionButtonsContainer>
-                  <ActionButtons>
-                    <Button href="#TODO">
-                      <Icon name="heart" color="black" />
-                    </Button>
-                    <Button href="#TODO">
-                      <Icon name="share" color="black" />
-                    </Button>
-                  </ActionButtons>
-                </ActionButtonsContainer>
-              </ControlsContainer>
-            </Container>
-          )
+      <Image
+        src={this.image.uri}
+        onClick={() => {
+          // tslint:disable-next-line:no-console
+          console.log("Zoom")
         }}
-      </Responsive>
+      />
+    )
+  }
+
+  renderImageArea(breakpoint?: string) {
+    const xs = breakpoint === "xs"
+    const ImageArea = xs ? SmallImageArea : LargeImageArea
+
+    return (
+      <ImageArea aspectRatio={this.image.aspectRatio}>
+        {this.hasMultipleImages && (
+          <Media greaterThan="xs">
+            <NavigationButton
+              direction="left"
+              onClick={this.changeCurrentImage.bind(this, -1)}
+            />
+          </Media>
+        )}
+
+        <ImageContainer>
+          <Media at="xs">{this.renderImage("xs")}</Media>
+          <Media greaterThan="xs">{this.renderImage()}</Media>
+        </ImageContainer>
+
+        {this.hasMultipleImages && (
+          <Media greaterThan="xs">
+            <NavigationButton
+              direction="right"
+              onClick={this.changeCurrentImage.bind(this, +1)}
+            />
+          </Media>
+        )}
+      </ImageArea>
+    )
+  }
+
+  renderControlsContainer(breakpoint?: string) {
+    const xs = breakpoint === "xs"
+    const ControlsContainer = xs
+      ? LargeControlsContainer
+      : SmallControlsContainer
+
+    return (
+      <ControlsContainer>
+        {this.hasMultipleImages && (
+          <PageIndicators
+            size={this.props.images.length}
+            highlightIndex={this.state.currentImage}
+            onSelect={i => this.setState({ currentImage: i })}
+          />
+        )}
+
+        <Media at="xs">
+          <Spacer />
+        </Media>
+
+        <ActionButtonsContainer>
+          <ActionButtons>
+            <Button href="#TODO">
+              <Icon name="heart" color="black" />
+            </Button>
+            <Button href="#TODO">
+              <Icon name="share" color="black" />
+            </Button>
+          </ActionButtons>
+        </ActionButtonsContainer>
+      </ControlsContainer>
+    )
+  }
+
+  render() {
+    return (
+      <Container>
+        <Media at="xs">
+          {this.renderImageArea("xs")}
+          {this.renderControlsContainer("xs")}
+        </Media>
+
+        <Media greaterThan="xs">
+          {this.renderImageArea()}
+          {this.renderControlsContainer()}
+        </Media>
+      </Container>
```

Comment:
(I'm gonna refactor all of these greaterThan='xs' at once in another commit)

Reply:
Not sure I understand what you were going to refactor here, can you elaborate?

Reply:
`greaterThan` -> `greaterThanOrEqual` (but i think this is already done)

Reply:
(Actually, gonna push that now)

Reply:
done!

Reply:
Ah gotcha. I actually think that when next to a `at="xs"` one it reads nice to use `greaterThan="xs"` but in standalone cases I agree that `greaterThanOrEqual` reads better. Anyways, no need to keep going back and forth on this right now.

Reply:
In that case I'm gonna pop off that last commit because I think it reads better alongside too.

Reply:
Hah, nice 👍